README.md

```diff
@@ -1,94 +1,87 @@
-# The ops library
+Ops is a Python framework for developing and testing charms. It is an official component of the Charm SDK, itself a crucial part of the Juju universe.
+||||
+|-|-|- |
+|| Learn about Juju! | Start deploying charms: <br> https://juju.is/docs/juju <br> _Juju is declarative, model-driven, and has native support for integrations and scaling, on any cloud. You want to deploy `foo`? Run `juju deploy foo`! You wish to integrate `foo` and `bar`? Run `juju integrate foo bar`!_ |
+||||
+|| Get better at Juju! | Take advantage of the charm ecosystem: <br> https://charmhub.io/ <br> _Quickly deploy a cluster ([OpenStack](https://charmhub.io/openstack-base), [Kubernetes](https://charmhub.io/charmed-kubernetes)), a database ([PostgreSQL](https://charmhub.io/postgresql-k8s), [MongoDB](https://charmhub.io/mongodb), etc.), an observability stack ([Canonical Observability Stack](https://charmhub.io/cos-lite)), an MLOps solution ([Kubeflow](https://charmhub.io/kubeflow)), and so much more!_ |
+||||
+|:point_right:| **Make Juju your own!** | Write your own charm: <br> https://juju.is/docs/sdk <br> _Juju is written in Go, but our SDK supports easy charm development in Python!_ |
 
-<!-- The text below is also at the top of ops/__init__.py. Keep in sync! -->
+## Use Ops
+Give it a quick try:
 
-The ops library is a Python framework ([`available on PyPI`](https://pypi.org/project/ops/)) for developing
-and testing [Juju](https://juju.is/) charms in a consistent way, using standard Python constructs
-to allow for clean, maintainable, and reusable code.
+_(assumes you have a Linux that supports installation via snap)_
 
-A charm is an operator -- business logic encapsulated in a reusable software
-package that automates every aspect of an application's life.
-
-Charms written with ops support Kubernetes using Juju's "sidecar charm"
-pattern, as well as charms that deploy to Linux-based machines and containers.
-
-Charms should do one thing and do it well. Each charm drives a single
-application and can be integrated with other charms to deliver a complex
-system. A charm handles creating the application in addition to scaling,
-configuration, optimisation, networking, service mesh, observability, and other
-day-2 operations specific to the application.
```

Comment:
What's the motivation for dropping this?

Reply:
Because this is defining what a charm _is supposed to be_, and that's something independent of Ops, so not really relevant here. 

We *could* make it relevant to Ops by using it to expand on the definition of Ops in the intro. E.g., instead of

> Ops is a Python framework for developing and testing charms.

we could say:

> Ops is a Python framework for developing and testing charms. It provides convenient Python wrappers around the hook tools that charm users can use to talk to the Juju API. 

However, I feel like that any elaboration of that sort makes more sense in the actual docs, so a better course of action would be to link to https://juju.is/docs/sdk/ops?dfg and, if the content there seems unsatisfactory, to iterate on it there. 

What do you think?